mainWindow.py

```diff
@@ -108,37 +115,47 @@ def on_runButton_clicked(self):
 
     @pyqtSlot()
     def on_configureButton_clicked(self):
-        # selectedIndices = self.sourceListView.selectedIndexes()
+        selectedIndices = self.sourceListView.selectedIndexes()
 
-        # if self.sourceModel.rowCount() is 0:
-        #     QMessageBox.warning(self, "No source directories",
-        #                         "There are no source directories in the list currently. Please add some folders "
-        #                         "before converting.")
-        #     return
-        # elif len(selectedIndices) == 0:
-        #     QMessageBox.warning(self, "No selected source directories",
-        #                         "There are no source directories selected currently. Please select one.")
-        #     return
-        # elif len(selectedIndices) != 1:
-        #     QMessageBox.warning(self, "Multiple selected directories",
-        #                         "There are currently more than one directories selected to configure. "
-        #                         "Please select only one.")
-        #     return
+        if self.sourceModel.rowCount() is 0:
+            QMessageBox.warning(self, "No source directories",
+                                "There are no source directories in the list currently. Please add some folders "
+                                 "before converting.")
+            return
+        elif len(selectedIndices) == 0:
+            QMessageBox.warning(self, "No selected source directories",
+                                "There are no source directories selected currently. Please select one.")
+            return
+        elif len(selectedIndices) != 1:
+            QMessageBox.warning(self, "Multiple selected directories",
+                                 "There are currently more than one directories selected to configure. "
+                                 "Please select only one.")
+            return
 
         # Get selected index text
-        # dataPath = selectedIndices[0].data()
-        dataPath = "C:/Users/Clint/PycharmProjects/SIUE-Dixon-Fat-Segmentation-Algorithm/MRI_Data/MF0302-PRE/"
+        dataPath = selectedIndices[0].data()
+        #dataPath = "/home/somecallmekenny/SIUE-Dixon-Fat-Segmentation-Algorithm/Data/Subject0001_Final/"
+
 
         # Load data from path
-        image, config = self.loadFile(dataPath)
-        if image is None:
-            return
+        # Use Correct Load File function based on datatype
+        DataTypeIndex = self.dataTypeComboBox.currentIndex() 
+        if DataTypeIndex == 0:
+            fatImage, waterImage, config = self.loadOldDixonFile(dataPath)
+            self.configureWindow = ConfigureWindow(fatImage, waterImage, config, dataPath, parent=self)
```

Comment:
This is good. I would rename ConfigureWindow to ConfigureWindowV1...this leaves room for ConfigureWindowV2..V3.